apps/native-component-list/webpack/webpack.prod.js

```diff
@@ -0,0 +1,148 @@
+const merge = require('webpack-merge');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const WorkboxPlugin = require('workbox-webpack-plugin');
+const webpack = require('webpack');
+const CompressionPlugin = require('compression-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const common = require('./webpack.common.js');
+const locations = require('./webpackLocations');
+
+const appEntry = [locations.appMain];
+
+const usePolyfills = false;
+
+if (usePolyfills) {
+  appEntry.unshift('babel-polyfill');
+}
+
+module.exports = merge(common, {
+  mode: 'production',
+  entry: {
+    vendor: ['react', 'react-native-web'],
+    app: appEntry,
+  },
+  devtool: 'hidden-source-map',
+  plugins: [
+    new CleanWebpackPlugin([locations.production.folder]),
+    new webpack.optimize.OccurrenceOrderPlugin(),
+    new webpack.optimize.ModuleConcatenationPlugin(),
+    new CopyWebpackPlugin([
+      {
+        from: locations.template.manifest,
+        to: locations.production.manifest,
+      },
+      {
+        from: locations.template.serveJson,
+        to: locations.production.serveJson,
+      },
+    ]),
+    new MiniCssExtractPlugin({
+      filename: 'static/css/[name].[contenthash].css',
+      chunkFilename: 'static/css/[name].[contenthash].chunk.css',
+    }),
+
+    // Moment.js is an extremely popular library that bundles large locale files
+    // by default due to how Webpack interprets its code. This is a practical
+    // solution that requires the user to opt into importing specific locales.
+    // https://github.com/jmblog/how-to-optimize-momentjs-with-webpack
+    // You can remove this if you don't use Moment.js:
+    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+    new CompressionPlugin({
+      test: /\.(js|css)$/,
+      filename: '[path].gz[query]',
+      algorithm: 'gzip',
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
+        loader: require.resolve('url-loader'),
+        options: {
+          limit: 10000,
```

Comment:
Set this smaller, here's a proposal:

```suggestion
          // Inline resources as Base64 when there is less reason to parallelize their download. The
          // heuristic we use is whether the resource would fit within a TCP/IP packet that we would
          // send to request the resource.
          //
          // An Ethernet MTU is usually 1500. IP headers are 20 (v4) or 40 (v6) bytes and TCP
          // headers are 40 bytes. HTTP response headers vary and are around 400 bytes. This leaves
          // about 1000 bytes for content to fit in a packet.
          limit: 1000,
```